lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -340,6 +354,8 @@ export class FormFieldModel extends FormWidgetModel {
             if (entry.length > 0) {
                 value = entry[0].id;
             }
+
+            return value;
```

Comment:
once again, usually you have a temp variable (value) to do different things in different places and 1 single return at the end. If you want to short-cut this by returning a soon as possible ( which sounds correct ) you don't need a temp variable u can just return what is needed for example 
 
```suggestion
            if (entry.length > 0) {
                return entry[0].id;
            }
```

Am I wrong or missing something?

Reply:
Yep, exacly I just miss this